xpublish_wms/cf_wms_router.py

```diff
@@ -197,7 +167,28 @@ def get_capabilities(ds: xr.Dataset, request: Request):
 
         # Not sure if this can be copied, its possible variables have different extents within
         # a given dataset probably, but for now...
-        bounding_box_element = ET.SubElement(layer, 'BoundingBox', attrib=bounds)
+        # Some basic check for dataset
+        da = ensure_crs(da)
+        if da.rio.crs == "EPSG:4326":
+            # Same CRS, don't need to rio.transform and prevent float round
+            west_lon = str(da.cf.coords["longitude"].min().values.item())
+            south_lat = str(da.cf.coords["latitude"].min().values.item())
+            east_lon = str(da.cf.coords["longitude"].max().values.item())
+            north_lat = str(da.cf.coords["latitude"].max().values.item())
+        else:
+            west_lon, south_lat, east_lon, north_lat = [str(round(v, 3)) for v in da.rio.transform_bounds(dst_crs="EPSG:4326", recalc=True)]
```

Comment:
I tried to use rioxarray for computing the bounds, but I gave me some weird rounding values

Somehow we would need it to offer specific `BoundinxBox` for each CRS .. but is it needed ?

Reply:
no, I am happy to force latlng for the bounds of the dataset for now. We can adjust later if necessary but its a corner case because even non regular grids are expected to have latlng information at this moment. Also currently the GetMap handler only supports 4326 and 3857 for now anyways.